pkgs/applications/editors/vim/plugins/non-generated/vectorcode-nvim/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  vimUtils,
+  vectorcode,
+  vimPlugins,
+}:
+let
+  inherit (vectorcode) src version;
+in
+vimUtils.buildVimPlugin {
+  inherit src version;
+
+  pname = "vectorcode.nvim";
+
+  sourceRoot = "${src.name}/plugin";
+
+  dependencies = [
+    vectorcode
+    vimPlugins.plenary-nvim
+  ];
+
+  buildInputs = [ vectorcode ];
+
+  postPatch = ''
+    cp -r ../lua .
+  '';
+
+  meta = {
+    description = "Index and navigate your code repository using vectorcode";
+    homepage = "https://github.com/Davidyz/VectorCode/blob/main/docs/neovim.md";
```

Comment:
```suggestion
    homepage = "https://github.com/Davidyz/VectorCode/blob/${version}/docs/neovim.md";
```

Reply:
@dotlambda I think you meant to replace `main` instead of `docs`.

In any case, my opinion is that people should be using `src.rev` for github blob urls. Though it doesn't really matter IMO: the point is that it should work in one way or another.

But yeah, inheriting from the original package is fine too.

Reply:
Please take a look at my latest push and let me know what you think